prover/prover_dal/src/fri_witness_generator_dal.rs

```diff
@@ -352,23 +353,23 @@ impl FriWitnessGeneratorDal<'_, '_> {
             sqlx::query!(
```

Comment:
please add a larger comment at the beginning of this function ('create_aggregation_jobs') - to better explain what it does.